### DISCUSSION TITLE
g100 does not contain $notification-background-error

### DISCUSSION THREAD

**Discussion by gcattan** [carbon-design-system/carbon#13651] [Help] [answered]
Hello, 

I have imported g100 theme like this:
`import { g100 } from '@carbon/themes';`

and I want to override the $notification-background-error token, but there is no such token in g100.

I am missing something?

**Comment by tw15egan** [comment, marked as answer]
Notification tokens emit their own set of tokens, so you would need to do something like this:

```scss
@use '@carbon/react/scss/themes';
@use '@carbon/react/scss/components/notification/tokens' with (
  $notification-background-error: (
    fallback: #3f51b5,
    values: (
      (theme: themes.$white, value: #3f51b5),
      (theme: themes.$g10, value: #d55bff),
      (theme: themes.$g90, value: #d0b2ff),
      (theme: themes.$g100, value: #cfd2ff),
    ),
  )
);

@use '@carbon/react/scss/reset';
@use '@carbon/react/scss/components/notification';
```


See more here under [component tokens](https://github.com/carbon-design-system/carbon/blob/acead26eeb6f15ec6d39970e4af08aa23ee48cdc/packages/styles/docs/sass.md)

**Comment by gcattan** [comment]
Perfect, thank you very much!
